osfoffline/settings/app_settings.py

```diff
@@ -23,6 +24,10 @@
     API_BASE = 'https://staging-api.osf.io'
     FILE_BASE = 'https://staging-files.osf.io'
 
+# if PROD_MODE:  # TODO: uncomment these before release
```

Comment:
There variables aren't really needed. When we go to production, we'll just change the non-dev-mode variables to point to production directly.

(Production vs dev mode won't be configurable in the deployed executable)